pkg/frontend/openshiftcluster_preflightvalidation_test.go

```diff
@@ -20,6 +21,58 @@ import (
 func TestPreflightValidation(t *testing.T) {
 	ctx := context.Background()
 	mockSubID := "00000000-0000-0000-0000-000000000000"
+	preflightPayload := []byte(`
```

Comment:
To DRY this up a bit, we could parameterize some of these mock IDs (such as `00000000-0000-0000-0000-000000000000`) with fmt.Sprintf - here's an example in another test: https://github.com/Azure/ARO-RP/blob/master/pkg/operator/controllers/machine/machine_controller_test.go#L243-L254